cmd/main.go

```diff
@@ -71,9 +77,41 @@ func init() {
 	if metrics.AreMetricsExposed(cf) {
 		metrics.InitializePrometheusMetrics()
 	}
+
+	mgr, err = ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
```

Comment:
we had better not move these initialization code in init function, we should keep it in the main function.